### DISCUSSION TITLE
snapserver with fft

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-snapclient#33] [Q&A] [answered]
i want to use your led fft example with snap, is it possible? could someone show me a brief example to get me along my way? im currently playing with esp32 tasks and putting copier.copy function into it!

**Comment by pschatzmann** [comment, marked as answer]
I don't see why this is not possible: just replace the i2s with the fft as destination.
If you want to send to output to multiple destinations (i2s and fft), use a MultiOutput
